docs/utilities/icons.md

```diff
@@ -362,39 +362,39 @@ All icons that are currently available for use
         <p class="icon-icon-eye"></p>
```

Comment:
iIs the class name really `.icon-icon-eye`? Because the provided example in the `<code>` has changed? Also didn't we want to replace the .svgs for the eye-icons? So these classes should already be in the icons.md.